msal4j-sdk/pom.xml

```diff
@@ -157,6 +157,12 @@
             <version>2.14.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
Is this new dependency used anywhere? Doesn't seem like there are new imports with it, and we're trying to remove dependencies anyway.